packages/merge-on-green/test/merge-on-green.ts

```diff
@@ -960,6 +1226,7 @@ describe('merge-on-green', () => {
 
       //This function is supposed to respond with an error
       it('does not add a PR if branch protection errors and comments on PR', async () => {
+        addPRStub.restore();
```

Comment:
one thought here, I think we could use Sinon's sandbox functionality and just always restart the sandbox in a `beforeEach`?

Reply:
I think we are, line 852.

We need to restore sometimes in order to provide specific functionality for some specific tests.

Reply:
If you need different setup scenarios, you can split describe blocks with separate beforeEach implementations.